PowerEditor/src/NppCommands.cpp

```diff
@@ -3793,8 +3793,8 @@ void Notepad_plus::command(int id)
 			}
 			else
 			{
-				_mainEditView.execute(SCI_RESETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, NULL);
-				_subEditView.execute(SCI_RESETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, NULL);
+				_mainEditView.execute(SCI_RESETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, 0);

```

Comment:
For WPARAM and LPARAM prefer "zero" instead of `NULL`. MS documentation is using mostly zero for them.